protos/local/temporal/sdk/core/bridge/service.proto

```diff
@@ -31,7 +31,7 @@ message StreamRequest {
 
   oneof request {
     InitRequest init = 2;
```

Comment:
This has changed too. We can just remove this entire file if it's not worth keeping updated since nothing implements or uses it.

Reply:
Done